### DISCUSSION TITLE
Bad username or password

### DISCUSSION THREAD

**Discussion by don-ferris** [kanboard/kanboard#5483] [Installation & Configuration]
### Checklist

- [x] I verified that Kanboard is correctly installed
- [x] I verified that the problem does not come from a plugin
- [x] I verified that the problem is not already reported
- [x] I understand that Kanboard is in maintenance mode. It doesn't mean it's abandoned, but there is no significant feature development

### Actual behaviour
Can't login - error message, "Bad username or password". Yesterday, I could login just fine. Nothing has changed. Using Bitwarden so it's not a caps lock/typo error.

### Configuration

- Kanboard version: not sure, can't login
- Database type and version: not sure
- PHP version:
- OS:
- Browser:

**Comment by don-ferris** [comment]
Tried ./cli user:reset-password. It comes back "user not found". How can I list users?

**Reply by fguillot** [reply]
> Tried ./cli user:reset-password. It comes back "user not found". How can I list users?

Connect to the database and look at the `users` table using SQL: `SELECT username FROM users;`.

**Reply by don-ferris** [reply]
How do I connect to the database?

**Comment by don-ferris** [comment]
Since I couldn't really get any help on this, I started over from scratch - this time creating just a single project in case it happened again. And it did. I guess I'm just DONE with Kanboard.

**Comment by alizehctl** [comment]
I've been facing this problem for a really long time. Resetting the password hasn't worked. I've tried different types of passwords as well. Right now only logging in via oauth works.
